skll/utilities/compute_eval_from_predictions.py

```diff
@@ -39,7 +99,26 @@ def compute_eval_from_predictions(examples_file, predictions_file,
     pred = {}
     with open(predictions_file) as pred_file:
         reader = csv.reader(pred_file, dialect=csv.excel_tab)
-        next(reader)  # skip header
+        header = next(reader)
+
+        # If prediction file contains probabilities instead of predictions, convert
+        # class probabilities to a class prediction using the specified
+        # `prediction_mathod`.
+        if probabilities:
+            classes = [c for c in header[1:] if c]
+            if prediction_method == 'expected_value':
+                if any(not c.isdigit() for c in classes):
```

Comment:
Why not just try to convert everything to `int` and catch a `TypeError` and then raise the exception?

Reply:
I considered both options, and favored asking permission over forgiveness, as I think it is easier to read. But I suppose trying to convert the classes to integers and catching the error is more robust.